docs/gallery/interactive/pyfar_audio_objects.ipynb

```diff
@@ -0,0 +1,412 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Intuitively, the `time` property can be used to access data of `TimeData` objects and the `freq` property to access data of `FrequencyData` objects. For `Signal` objects time and frequency data can be accessed at any time and the Signal class converts between the time and frequency domain if required using the Discrete Fourier Transform. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Accessing frequency data from the signal that was created with time data\n",
+    "signal.freq"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "At this point it is **important to note** that the data returned by the `freq` property can depend on the normalization of the Discrete Fourier Transform as explained in more detail `here`. The frequency data without any normalization can be accessed using the `freq_raw` property. However, for this example the two return the same results"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "signal.freq_raw"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Read on to learn how to find out which normalization of the Discrete Fourier Transform is used for a signal."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Write data to audio objects\n",
+    "\n",
+    "The same properties can be used to set the data inside audio objects."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "signal.time = [2, 0, 0, 0]\n",
+    "print(f'signal.time = {signal.time}')\n",
+    "signal2.freq = [2, 2, 2]\n",
+    "print(f'signal2.freq = {signal2.freq}')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Additional information stored in audio objects\n",
+    "\n",
+    "Converting between time and frequency data is already useful, but audio objects can do more. They contain additional information about their data that can be helpful. Some basic information is already returned when prompting an audio object"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "signal"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Lets break down the information one by one. \n",
+    "\n",
+    "**Domain:** The signal is in the time domain, because we last accessed its time data above. We can get and set this property with the `signal.domain` attribute but it is of not much interested when working with Signal objects because they are automatically converted between the time and frequency domain, when it is required."
```

Comment:
```suggestion
    "**Domain:** The signal is in the time domain, because we last accessed its time data above. We can get and set this property with the `signal.domain` attribute but it is of not much interest when working with Signal objects because they are automatically converted between the time and frequency domain, when it is required."
```